packages/quantic/force-app/main/default/lwc/quanticPager/e2e/quanticPager.e2e.ts

```diff
@@ -0,0 +1,99 @@
+import {testSearch, testInsight, expect} from './fixture';
+import {useCaseTestCases} from '../../../../../../playwright/utils/useCase';
+
+const numberOfResultsPerPage = 10;
+
+const fixtures = {
+  search: testSearch,
+  insight: testInsight,
+};
+
+useCaseTestCases.forEach((useCase) => {
+  let test = fixtures[useCase.value];
+
+  test.describe(`quantic pager ${useCase.label}`, () => {
+    test.describe('when clicking the next page button', () => {
+      test('should trigger a new search and log analytics', async ({
+        pager,
+        search,
+      }) => {
+        const searchResponsePromise = search.waitForSearchResponse();
+        await pager.clickNextPageButton();
+        const searchResponse = await searchResponsePromise;
+        const {firstResult} = searchResponse.request().postDataJSON();
+        expect(firstResult).toBe(numberOfResultsPerPage);
+        await pager.waitForPagerNextUaAnalytics();
+      });
+    });
+
+    test.describe('when clicking the previous page button', () => {
+      test('should trigger a new search and log analytics', async ({
+        pager,
+        search,
+      }) => {
+        await pager.clickNextPageButton();
+        await search.waitForSearchResponse();
+
+        const searchResponsePromise = search.waitForSearchResponse();
+        await pager.clickPreviousPageButton();
+        const searchResponse = await searchResponsePromise;
+        const {firstResult} = searchResponse.request().postDataJSON();
+        expect(firstResult).toBe(0);
+        await pager.waitForPagerPreviousUaAnalytics();
+      });
+    });
+
+    test.describe('when clicking a specific page button', () => {
+      test('should trigger a new search and log analytics', async ({
+        pager,
+        search,
+      }) => {
+        const examplePage = 3;
+        const searchResponsePromise = search.waitForSearchResponse();
+        await pager.clickPageNumberButton(examplePage);
+        const searchResponse = await searchResponsePromise;
+        const {firstResult} = searchResponse.request().postDataJSON();
+        expect(firstResult).toBe(numberOfResultsPerPage * (examplePage - 1));
+        await pager.waitForPagerNumberUaAnalytics();
+      });
+    });
+
+    test.describe('when a new search is made', () => {
+      test('should select the first page', async ({pager, search}) => {
+        await pager.clickNextPageButton();
+        await search.waitForSearchResponse();
+
+        await expect(pager.previousPageButton).not.toBeDisabled();
+        const searchResponsePromise = search.waitForSearchResponse();
+        await search.performSearch();
+        const searchResponse = await searchResponsePromise;
+        const {firstResult} = searchResponse.request().postDataJSON();
+
+        const expectedPage = 0;
+        await expect(pager.pageButtons.nth(expectedPage)).toHaveAttribute(
+          'aria-pressed',
+          'true'
+        );
+        expect(firstResult).toBe(0);
```

Comment:
I'm guessing you meant to do this?

```suggestion
        expect(firstResult).toBe(expectedPage);
```